Woof/Woof/ViewModel/SitterProfileViewModel.swift

```diff
@@ -109,6 +113,16 @@ final class SitterProfileViewModel: ObservableObject {
         }
     }
 
+    private func update(_ sitter: Sitter) async throws {
+        let endpoint = WoofAppEndpoint.updateSitter(sitter.asDictionary())
+
+        do {
+            _ = try await NetworkService<WoofAppEndpoint>().request(endpoint)
```

Comment:
as we use an instance of `NetworkService` twice does it make sense to create a private property for it
` private let service = NetworkService<WoofAppEndpoint>()`